src/vulkan/vk-pipeline.cpp

```
@@ -89,14 +92,16 @@ Result serializePipelineBinaries(DeviceImpl* device, VkPipeline pipeline, ISlang
 
     VkPipelineBinaryHandlesInfoKHR binaryHandlesInfo = {VK_STRUCTURE_TYPE_PIPELINE_BINARY_HANDLES_INFO_KHR};
 
-    SLANG_VK_RETURN_ON_FAIL(
-        api.vkCreatePipelineBinariesKHR(device->m_device, &binaryCreateInfo, nullptr, &binaryHandlesInfo)
+    SLANG_VK_RETURN_ON_FAIL_REPORT(
```

Cppcheck (2.20.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)